README.md

```diff
@@ -32,7 +32,8 @@ Download the appropriate Julia version for your system from [here](https://julia
 ### Install the PiCraft package
 
 ```julia
-Pkg.clone("https://github.com/JuliaBerry/PiCraft.jl")
+using Pkg
+Pkg.add("https://github.com/JuliaBerry/PiCraft.jl")
```

Comment:
We should probably add this to METADATA now?

Reply:
So this was already in the registry, just needed a new release to be tagged. The installation should be `Pkg.add("PiCraft")`

Reply:
```
(v1.0) pkg> add PiCraft
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package PiCraft [6cc4b21d]:
 PiCraft [6cc4b21d] log:
 ├─possible versions are: 0.1.0 or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions 0.1.0
 └─restricted by julia compatibility requirements to versions: uninstalled — no versions left
```
Is this due to the REQUIRE file ? REQUIRE currently states 0.6 instead of 0.7 or 1.0. Or is there some other reason behind this ?